src/zepben/evolve/database/sqlite/tables/metadata_tables.py

```diff
@@ -13,7 +13,7 @@
 class TableVersion(SqliteTable):
     version: Column = None
 
-    SUPPORTED_VERSION = 37
+    SUPPORTED_VERSION = 41
```

Comment:
This version needs to match what is in the JVM version for the database to actually work. 
* change set 38: Already implemented, this is what the SUPPORTED_VERSION should have been set to previously.
* change set 39: This is what the SUPPORTED_VERSION should be set to with the changes currently in this PR.
* change set 40: Changes to indexes which have not been included in this PR. Search for EWB-1043 and include the change. This is the SUPPORTED_VERSION this should ultimately be set to.

Reply:
@kgreav asked me to change it to 41, "anticipating another change going in before yours that will do 40". Should I set it to 39 or include EWB-1043 and set it to 40?

Reply:
If you want to keep things in seperate PR's set it to 39 and then do the required changes for the 40 changeset in another PR, otherwise just merge the 40 change set into this one to save on waiting for actions to run